src/docs/product/data-management-settings/dynamic-sampling/index.mdx

```diff
@@ -44,9 +44,9 @@ While prioritizing data from the latest release, we also take into consideration
 
 We prioritize data from development environments because these environments typically generate a relatively small number of transactions compared to your production environments. As a result, to gain meaningful insights during your testing phase, we prioritize transaction data from these environments.
 
-### Key Transactions
+### Starred Transactions
 
-Typically, for most projects there are only a few transactions that you really care about. We prioritize [key transactions](/product/performance/transaction-summary/#starring-key-transactions) over others, so you can have more visibility into your important data.
+Typically, for most projects there are only a few transactions that you really care about. We prioritize [starred transactions](/product/performance/transaction-summary/#starring-key-transactions) over others, so you can have more visibility into your important data.
```

Comment:
it would be nice to update the screenshot too

Reply:
Agreed, we'll want to update the screenshot before pushing this change.